managers/GestionGlobal.java

```diff
@@ -0,0 +1,39 @@
+package managers;
+
+import utility.FileManager;
+
+import java.util.ArrayList;
+
+public abstract class GestionGlobal {
```

Comment:
- [ ] TODO: changer le nom?

```suggestion
public abstract class Manager{
```